learn/what_is_meilisearch/telemetry.md

```diff
@@ -174,3 +170,6 @@ This list is liable to change with every new version of Meilisearch. It's not be
 | `typo_tolerance.min_word_size_for_typos.two_typos` | The defined value for the `minWordSizeForTypos.twoTypos` parameter                          | 9
 | `pagination.max_total_hits`                        | The defined value for the `pagination.maxTotalHits` property                                | 1000
 | `faceting.max_values_per_facet`                    | The defined value for the `faceting.maxValuesPerFacet` property                             | 100
+| `per_index_uid`                                    | `true` if the `uid` is used to fetch an index stat resource, otherwise `false`              | `false`
```

Comment:
```suggestion
| `per_index_uid`                                    | `true` if the `uid` is used to fetch an index stat resource, otherwise `false`              | false
```